requirements.txt

```diff
@@ -1,3 +1,5 @@
+django<1.9,>=1.8
```

Comment:
It needs to be a versioned django in requirements.txt or we repeat the line three times in tox.ini